cmd/api/api_server.go

```diff
@@ -22,9 +24,25 @@ var (
 	rdb      *redis.Client
 	ctx      context.Context
 	redisKey = "Stock"
+	logger   *zap.Logger
 )
 
+// ClusterListResponse models the Cluster list response returned by this API
+type ClusterListResponse struct {
+	Count    int                 `json:"count"`
+	Clusters []inventory.Cluster `json:"clusters"`
+}
+
+// AccountListResponse model the Accounts list response returned by this API
```

Comment:
WDYT about?
// AccountListResponse represents the API response containing a list of accounts